test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/session/DefaultSessionBuilderInstantiator.java

```diff
@@ -16,13 +16,18 @@
 package com.datastax.oss.driver.api.testinfra.session;
 
 import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.config.DriverConfigLoaderBuilder;
 import com.datastax.oss.driver.api.core.session.SessionBuilder;
 
 public class DefaultSessionBuilderInstantiator {
   public static SessionBuilder<?, ?> builder() {
     return CqlSession.builder();
   }
 
+  public static DriverConfigLoaderBuilder configLoaderBuilder() {
```

Comment:
Changing our tests to use this instead of `TestConfigLoader`, we can get rid of `configPath()`, and then this class starts to look at a lot like `CqlSession`.  In fact, we can get rid of `DefaultSessionBuilderInstantiator` completely and just point to `CqlSession` instead.